xcp_d/utils/fcon.py

```diff
@@ -4,12 +4,22 @@
 import nibabel as nb
 import numpy as np
 from nilearn.input_data import NiftiLabelsMasker
+from nipype import logging
 from scipy import signal
 from scipy.stats import rankdata
 from templateflow.api import get as get_template
 
+LOGGER = logging.getLogger("nipype.utils")
 
-def extract_timeseries_funct(in_file, mask, atlas, timeseries, fconmatrix):
+
+def extract_timeseries_funct(
```

Comment:
I'd be ok with renaming this function

Reply:
Definitely a good idea. I started renaming it and then decided to just move the meat of the function into the NiftiConnect interface, since that's the only place the function is used.